content/en/docs/concepts/_index.md

```diff
@@ -1,12 +1,10 @@
 ---
-title: Concepts
-description: >-
-  What is OpenTelemetry, what does it provide and what does it support?
-spelling: cSpell:ignore otel
-aliases: [/about, /docs/concepts/overview, /otel]
```

Comment:
Note that the two aliases `/about` and `/otel` are now associated with the "What is OpenTelemetry?" page